easybuild/easyblocks/m/mumps.py

```diff
@@ -61,7 +62,11 @@ def configure_step(self):
             optf = "-Dintel_ -DALLOW_NON_INIT -nofor-main"
             optl = "-nofor-main"
         elif comp_fam == toolchain.GCC:  #@UndefinedVariable
-            make_inc_templ = 'Makefile.gfortran.%s'
+            if LooseVersion(self.version) >= LooseVersion("5.0.0"):
+                make_inc_templ = 'Makefile.debian.%s'
```

Comment:
Apparently MUMPS have decided to change the name of the makefile templates and Makefile.gfortran is not longer in there. The equivalent is Makefile.debian